packages/frontend/src/features/sqlRunner/components/Header.tsx

```diff
@@ -94,6 +97,22 @@ export const Header: FC = () => {
                         >
                             <ShareSqlLinkButton />
                         </Tooltip>
+                        <Tooltip
```

Comment:
This should be hidden if the chart isn't saved 
Perhaps we can check if there's a `savedSqlUuid`?